### PR TITLE
Add actions

### PR DESCRIPTION
**Pull request by iampeternguyen**
This PR adds the menuItem to the protoplugin. 

To use this, in open project core

import menuItem in the `wp-view-context-menu.directive.ts`

and then in the buildItems` function, before items is returned. add `items.push(menuItem)`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,9 @@ In the following sections we will explain some common features that you may want
 
 Each section will list the relevant files you may want to look at and explain the features. Beyond that there are also code comments in the respective files which provide further details.
 
+### Plugin Path in the OpenProject core App
+the files in frontend repository imports other modules in the core app with the `core-app/` prefix which is an alias pointing to `<core-app-root>/frontend/src/app` defined in the `tsconfig.base.json` file, be careful to update import path when configurations change
```
Comment (tessi) [inline README.md:67]:
```suggestion

JS files in `/frontend` import other modules in the core app with the `core-app/` prefix which is an alias pointing to `<core-app-root>/frontend/src/app` defined in the `tsconfig.base.json` file, be careful to update import path when configurations change.
```

just some typo fixing and minor (subjective?) readability improvements :)

Comment (iampeternguyen) [reply README.md:67]: closing this. I realized i was pushing to the wrong repo and not our forked one.

**Comment by oliverguenther** [comment]
Hi @iampeternguyen , If you won't mind I'll include your changes to the Readme. Also, why not add the action through the hook service like with costs? https://github.com/opf/openproject/blob/dev/modules/costs/frontend/module/main.ts#L38-L58

Cheers
Oliver

**Comment by iampeternguyen** [comment]
> Hi @iampeternguyen , If you won't mind I'll include your changes to the Readme. Also, why not add the action through the hook service like with costs? https://github.com/opf/openproject/blob/dev/modules/costs/frontend/module/main.ts#L38-L58
> 
> Cheers Oliver

Sure thing. There are also some updates to the the references to core-app references in the kitten components that would help fix some errors.

And yeah! i just discovered that as well. I'm brand new to working with OP so there's lots of stuff i don't quite understand. I am able to see that my action is called from the hook service, but it won't get added to the menu because it's not an allowed action. I'm still trying to figure out how to add register the action with the workpackage so that it can pass the `workPackage.hasOwnProperty(action.link)` check. Is there another place that you could point me to so I can try to figure that out?

**Pull request closed without merging**